pimd/pim_iface.h

```diff
@@ -94,16 +95,14 @@ struct pim_interface {
 	uint32_t options; /* bit vector */
 	ifindex_t mroute_vif_index;
 	struct pim_instance *pim;
-
-	struct in_addr primary_address; /* remember addr to detect change */
+	pim_addr primary_address;       /* remember addr to detect change */
 	struct list *sec_addr_list;     /* list of struct pim_secondary_addr */
-	struct in_addr update_source;   /* user can statically set the primary
+	pim_addr update_source;		/* user can statically set the primary
 					 * address of the interface */
-
-	int igmp_version;		       /* IGMP version */
-	int igmp_default_robustness_variable;  /* IGMPv3 QRV */
-	int igmp_default_query_interval;       /* IGMPv3 secs between general
-						  queries */
+	int igmp_version;		/* IGMP or MLD version */
+	int igmp_default_robustness_variable; /* IGMP or MLD QRV */
```

Comment:
this could be renamed to something which depicts both

Reply:
@g-balaji1: We are not changing the names in this PR. The names have been changed in other PRs.